### ISSUE TITLE
Lock re-login process flow stuck

### ISSUE THREAD

**Issue by lordakshaya** [auth0/lock#244]
After I have logged into my application once, when I open the app again - I get stuck on this page:

![image](https://cloud.githubusercontent.com/assets/1169337/10645470/119e9e04-7858-11e5-89b0-e8791a0e323d.png)

**Comment by rolodato** [comment]
Any JS errors or failed requests in dev tools? A sample project to reproduce the problem would also help.

**Comment by lordakshaya** [comment]
@rolodato:
Console Error - none

Response JSON structure: 

```
lastUsedClientID: "xxxxxxxxxxx"
lastUsedConnection: {name: "sms", strategy: "sms"}
lastUsedUserID: "sms|xxxxxxxxxx"
lastUsedUsername: "+0000000000"
sessionClients: ["xxxxxx"]
sso: true
```

Clicking on Show all, doesnt trigger anything.

**Comment by gnandretta** [comment]
@lordakshaya @rolodato We've released v7.10.2 which should fix the issue.

**Comment by rolodato** [comment]
@lordakshaya Just to give some context, this was being caused because Lock fetches the Auth0 SSO cookie to determine which connection was used by the user. Lock was not recognising `sms` as a valid connection, so it failed to display the dialog. 7.10.2 was updated to ignore unknown connections instead of failing.
